README.md

```diff
@@ -8,39 +10,25 @@
 
 *The graph database for delightful React state management*
 
-## Installation
-
-```bash
-# NPM
-npm install homebase-react --save
-
-# Yarn
-yarn add homebase-react
-```
-
-## Purpose
-
-Data should be a first class citizen on the client. We deserve the ergonomics and power of relational databases not just on the server, but in our React applications.
-
-Homebase React lets you plug a relational database into your React application with just 3 lines of code. In fact, it's the same database that powers Roam Research and many other ClojureScript applications, but with an API that's familiar to React and JS developers.
 
-- Replace Redux with something simpler and more declarative
-- Stop spending time wiring up custom datatypes, reducers, caches and other bespoke state mumbo jumbo
-- Gain all the benefits of relational data like a single source of truth
-- Traverse your data graph like it's one big JSON object
-- Query your database with a convenient JSON query syntax
-- Query your database with Clojure style [Datalog](https://docs.datomic.com/on-prem/query.html) if you need more power
-- It's just data
+Homebase React makes state management painless by enabling you to plug a relational graph database into your React application with just 3 lines of code. This is the same database that powers Roam Research and many other ClojureScript applications, but with an API that's familiar to React and JS developers.
 
-The goal of Homebase React is to be immediately more intuitive than any denormalized JSON store and over time we will eliminate all the tedious aspects of manipulating data on clients. Homebase React makes it easier to work with complex data on client by making our applications [local-first](https://news.ycombinator.com/item?id=21581444).
+## Features
+- The simplest and most declarative state management solution
+- The power of a backend relational graph database, but without having to wait on the network
+- Convenient JSON query syntax
+- Powerful Clojure style [Datalog](https://docs.datomic.com/on-prem/query.html) query syntax if you need it
+- Traverse your data graph like it's a big JSON object
+- Backup your data to the cloud
 
-## Examples
-### Live Demos
-You can see our hosted live demos [here](https://homebaseio.github.io/homebase-react)
-
-### Code Examples
-You can clone and run our React code examples [here](https://github.com/homebaseio/homebase-react-examples).
+## Roadmap
 
+1. Document integration with backends
```

Comment:
Since we just added Firebase change to

Document integration with more backends